Source/missiles.cpp

```diff
@@ -3332,7 +3331,7 @@ void MI_Search(Missile &missile)
 		return;
 
 	missile._miDelFlag = true;
-	PlaySfxLoc(IS_CAST7, Players[missile.var1].position.tile);
+	PlaySfxLoc(IS_CAST7, Players[missile._misource].position.tile);

```

Comment:
Technically this change causes the sound to always be played for player 0 if the game is loaded back in the original game... but that just happens to be the ide of the player in single player.